.travis.yml

```diff
@@ -17,3 +17,12 @@ deploy:
   on:
     repo: m-lab/m-lab.github.io
     branch: master
+before_install:
+  if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    openssl aes-256-cbc -K $encrypted_69b240b11ca3_key -iv $encrypted_69b240b11ca3_iv -in _cf_s3_invalidator.yml.enc -out _cf_s3_invalidator.yml -d;
```

Comment:
Can we add documentation for what this is doing and why it's necessary? I think the documentation you wrote in the PR description is good. I'd like it to live with the code, so can we adapt it and bring it in here?

Similarly to the S3 key, can we include instructions in the README for how to regenerate this data if we need to roll creds?